examples/configures/ACNN_AC/PDBbind_R2.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Is this file used anywhere?

Reply:
It isn't currently used, but it could be used to re-run the benchmark with optimal hyperparams.